brayns/engine/Model.cpp

```diff
@@ -421,11 +421,10 @@ void Model::updateSizeInBytes()
         _sizeInBytes += sdfNeighbours.size() * sizeof(size_t);
 }
 
-void Model::_updateBounds()
+void Model::updateBounds()
 {
     if (_spheresDirty)
     {
-        _spheresDirty = false;
```

Comment:
why removed here and repeating this in every engine?

Reply:
I created a common method for marking geometries clean.